README.md

```diff
@@ -187,12 +187,13 @@ When updating this, also update:
 
 Tokio will keep a rolling MSRV (minimum supported rust version) policy of **at
 least** 6 months. When increasing the MSRV, the new Rust version must have been
-released at least six months ago. The current MSRV is 1.56.0.
+released at least six months ago. The current MSRV is 1.63.0.
 
 Note that the MSRV is not increased automatically, and only as part of a minor
 release. The MSRV history for past minor releases can be found below:
 
- * 1.27 to now - Rust 1.56
+ * 1.30 to now - Rust 1.63
+ * 1.27 to 1.29.1 - Rust 1.56
```

Comment:
This list only specifies the minor release, because patch releases never bump the MSRV.
```suggestion
 * 1.27 to 1.29 - Rust 1.56
```

Reply:
Thanks!
I've fixed this.